modules/fb_apache/configuration/kibana/search/Apache2-access-logs.json

```diff
@@ -0,0 +1,19 @@
+{
+  "sort": [
+    "@timestamp", 
+    "desc"
+  ], 
+  "hits": 0, 
+  "description": "", 
+  "title": "Apache2 access logs", 
+  "version": 1, 
+  "kibanaSavedObjectMeta": {
+    "searchSourceJSON": "{\"index\":\"filebeat-*\",\"query\":{\"query_string\":{\"query\":\"_exists_:apache2.access\",\"analyze_wildcard\":true}},\"filter\":[],\"highlight\":{\"pre_tags\":[\"@kibana-highlighted-field@\"],\"post_tags\":[\"@/kibana-highlighted-field@\"],\"fields\":{\"*\":{}},\"require_field_match\":false,\"fragment_size\":2147483647}}"
```

Comment:
I had the `Could not locate that search` error even when the index name was still `filebeat-*`, though you are correct that the index name will cause a fail.  The problem I had was that saved searches aren't all being uploaded properly.  @guyboertje has a fix for that coming soon.